include/tables/scene_table.h

```diff
@@ -9,107 +9,107 @@
+/* 0x0F */ DEFINE_SCENE(ganontikasonogo_scene, none, SCENE_GANONTIKA_SONOGO, SDC_GANONTIKA_SONOGO, 0, 0) // INSIDE_GANONS_CASTLE_COLLAPSE
+/* 0x10 */ DEFINE_SCENE(takaraya_scene, g_pn_51, SCENE_TAKARAYA, SDC_DEFAULT, 0, 0) // TREASURE_BOX_SHOP
+/* 0x11 */ DEFINE_SCENE(ydan_boss_scene, none, SCENE_YDAN_BOSS, SDC_YDAN_BOSS, 0, 0) // DEKU_TREE_BOSS
+/* 0x12 */ DEFINE_SCENE(ddan_boss_scene, none, SCENE_DDAN_BOSS, SDC_DEFAULT, 0, 0) // DODONGOS_CAVERN_BOSS
+/* 0x13 */ DEFINE_SCENE(bdan_boss_scene, none, SCENE_BDAN_BOSS, SDC_BDAN, 0, 0) // JABU_JABU_BOSS
+/* 0x14 */ DEFINE_SCENE(moribossroom_scene, none, SCENE_MORIBOSSROOM, SDC_DEFAULT, 1, 0) // FOREST_TEMPLE_BOSS
+/* 0x15 */ DEFINE_SCENE(FIRE_bs_scene, none, SCENE_FIRE_BS, SDC_HIDAN, 0, 0) // FIRE_TEMPLE_BOSS
+/* 0x16 */ DEFINE_SCENE(MIZUsin_bs_scene, none, SCENE_MIZUSIN_BS, SDC_MIZUSIN_BS, 0, 0) // WATER_TEMPLE_BOSS
+/* 0x17 */ DEFINE_SCENE(jyasinboss_scene, none, SCENE_JYASINBOSS, SDC_DEFAULT, 0, 0) // SPIRIT_TEMPLE_BOSS
+/* 0x18 */ DEFINE_SCENE(HAKAdan_bs_scene, none, SCENE_HAKADAN_BS, SDC_HAKADAN, 0, 0) // SHADOW_TEMPLE_BOSS
+/* 0x19 */ DEFINE_SCENE(ganon_boss_scene, none, SCENE_GANON_BOSS, SDC_DEFAULT, 0, 0) // BOSS_GANONDORF
+/* 0x1A */ DEFINE_SCENE(ganon_final_scene, none, SCENE_GANON_FINAL, SDC_GANON_FINAL, 0, 0) // GANONS_CASTLE_COLLAPSE
+/* 0x1B */ DEFINE_SCENE(entra_scene, none, SCENE_ENTRA, SDC_DEFAULT, 0, 0) // MARKET_ENTRANCE_DAY
+/* 0x1C */ DEFINE_SCENE(entra_n_scene, none, SCENE_ENTRA_N, SDC_DEFAULT, 0, 0) // MARKET_ENTRANCE_NIGHT
+/* 0x1D */ DEFINE_SCENE(enrui_scene, none, SCENE_ENRUI, SDC_DEFAULT, 0, 0) // MARKET_ENTRANCE_RUINS
+/* 0x1E */ DEFINE_SCENE(market_alley_scene, g_pn_18, SCENE_MARKET_ALLEY, SDC_DEFAULT, 0, 0) // BACK_ALLEY_DAY
+/* 0x1F */ DEFINE_SCENE(market_alley_n_scene, g_pn_18, SCENE_MARKET_ALLEY_N, SDC_DEFAULT, 0, 0) // BACK_ALLEY_NIGHT
+/* 0x20 */ DEFINE_SCENE(market_day_scene, g_pn_17, SCENE_MARKET_DAY, SDC_DEFAULT, 0, 0) // MARKET_DAY
+/* 0x21 */ DEFINE_SCENE(market_night_scene, g_pn_17, SCENE_MARKET_NIGHT, SDC_DEFAULT, 0, 0) // MARKET_NIGHT
+/* 0x22 */ DEFINE_SCENE(market_ruins_scene, g_pn_17, SCENE_MARKET_RUINS, SDC_DEFAULT, 0, 0) // MARKET_RUINS
+/* 0x23 */ DEFINE_SCENE(shrine_scene, none, SCENE_SHRINE, SDC_DEFAULT, 0, 0) // TEMPLE_OF_TIME_EXTERIOR_DAY
+/* 0x24 */ DEFINE_SCENE(shrine_n_scene, none, SCENE_SHRINE_N, SDC_DEFAULT, 0, 0) // TEMPLE_OF_TIME_EXTERIOR_NIGHT
+/* 0x25 */ DEFINE_SCENE(shrine_r_scene, none, SCENE_SHRINE_R, SDC_DEFAULT, 0, 0) // TEMPLE_OF_TIME_EXTERIOR_RUINS
 /* 0x26 */ DEFINE_SCENE(kokiri_home_scene, none, SCENE_KOKIRI_HOME, SDC_DEFAULT, 0, 0)
 /* 0x27 */ DEFINE_SCENE(kokiri_home3_scene, none, SCENE_KOKIRI_HOME3, SDC_DEFAULT, 0, 0)
 /* 0x28 */ DEFINE_SCENE(kokiri_home4_scene, none, SCENE_KOKIRI_HOME4, SDC_DEFAULT, 0, 0)
 /* 0x29 */ DEFINE_SCENE(kokiri_home5_scene, none, SCENE_KOKIRI_HOME5, SDC_DEFAULT, 0, 0)
 /* 0x2A */ DEFINE_SCENE(kakariko_scene, none, SCENE_KAKARIKO, SDC_DEFAULT, 0, 0)
```

Comment:
this one is a bit weird
gz has this as `CARPENTERS_HOUSE`, cause at night as child all of the carpenters and their boss are inside. But talon also stays in here as adult, which makes it seem more like an inn of somekind? the lady in here always seems to be just housing random people.

the main ladies dialogue is not too helpful. not sure if anyone knows anything else that is more definitive

Reply:
Kakariko guest house ?

Reply:
suppose so. dont like that its an assumption on our part cause nothing in the game says that it is an inn or guest house, but its how it appears to be based on usage

Reply:
We could also just kakariko_center_house it but idk

Reply:
I feel like "guest house" kinda works since carpenters and talon only stay there for some time, sounds like the shortest way to refer to that part of the lore
`KAKARIKO_GUEST_HOUSE`

Or refer to the house by its location somewhat in the center of the village
`KAKARIKO_CENTER_HOUSE` / `KAKARIKO_CENTRAL_HOUSE`

Or there's the fan-favourite "whatever this is idc"
`KAKARIKO_MISC_HOUSE`

Reply:
Changed to `kakariko_center_house` but open to discussion